data/human/free worlds 3 checkmate.txt

```diff
@@ -171,7 +171,7 @@ mission "FWC Cebalrai 1"
 		dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
 	
 	on offer
-		log "Captured the Kaus Borealis system from the Navy, even though the locals are still loyal to the Republic. Also, learned that the Wolf Pack ships have been using nuclear missiles, and that is why they were so effective in decimating the Navy fleet."
+		log "Captured the Kaus Borealis system from the Navy, even though the locals are still loyal to the Republic. Also, learned that the Wolf Pack ships have been using Nuclear Missiles, and that is why they were so effective in decimating the Navy fleet."
```

Comment:
I think that this is fine to be lowercase. While we do have a literal "Nuclear Missile" outfit, the concept of nuclear missiles is generic. (The only other missile that is in a similar boat is Torpedoes, but I'm fine with leaving that capitalized, although it might be worth looking for a proper name for them.)

Reply:
```suggestion
		log "Captured the Kaus Borealis system from the Navy, even though the locals are still loyal to the Republic. Also, learned that the Wolf Pack ships have been using nuclear missiles, and that is why they were so effective in decimating the Navy fleet."
```